assets/components/productPage/productPagePlanForm/productPagePlanFormLabel.scss

```diff
@@ -56,8 +56,8 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  width: $gu-v-spacing * 2;
-  height: $gu-v-spacing * 2;
+  width: $gu-v-spacing * 1.666;
```

Comment:
magic numbers 😄

Reply:
hahaha, i always try to round them a bit but yep, this is made to reach a min-height of 32px, could be improved!